topi/include/topi/transform.h

```diff
@@ -1106,7 +1106,7 @@ inline Tensor arange(const Expr start,
 inline Tensor layout_transform(const Tensor& src,
                                const std::string& src_layout,
                                const std::string& dst_layout,
-                               const std::string name = "layout_transform",
+                               const std::string name = "T_layout_trans",
```

Comment:
```suggestion
                               const std::string name = "T_layout_transform",
```

Reply:
For unambiguous long names, I would like abbreviations.